apps/web/components/dashboard/tags/AllTagsView.tsx

```diff
@@ -104,22 +187,73 @@ export default function AllTagsView({
   const tagsToPill = (tags: typeof allTags) => {
     let tagPill;
     if (tags.length) {
-      tagPill = tags.map((t) => (
-        <TagPill key={t.id} id={t.id} name={t.name} count={t.count} />
-      ));
+      tagPill = (
+        <div className="flex flex-wrap gap-3">
```

Comment:
It seems that this `div` is redundant?

<img width="301" alt="Screenshot 2024-05-16 at 10 31 42 PM" src="https://github.com/MohamedBassem/hoarder-app/assets/2418637/385eeefe-36d5-497c-83f3-8f39f5f84643">